Server/Components/PlayerMovement.cpp

```diff
@@ -200,6 +216,12 @@ void PlayerMovement::FireGrapple(Vector3 grapplePoint) {
 
 
 void PlayerMovement::Jump() {
-    gameObject->GetPhysicsObject()->AddForce(Vector3{0, 1, 0} * jumpForce);
+    hasCoyoteExpired = false;
+    PhysicsObject* physOb = gameObject->GetPhysicsObject();
```

Comment:
Does this pointer need deleting at the end of the func?

Reply:
No it doesnt, this is just getting a reference to the player's physics object: it has no ownership over it.